observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator by using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
```

Comment:
```suggestion
You can now use Prometheus metrics and rely less on Loki for storage with the Network Observability Operator. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
```
for minimalism and directness

Reply:
Thanks. I sometimes struggle with tending to be too "markety" in release notes.